package-configuration/linux-cli/test_get_cloudify.py

```diff
@@ -144,6 +147,25 @@ def test_check_cloudify_installed_in_venv(self):
         finally:
             shutil.rmtree(tmp_venv)
 
+    def test_get_requirements_from_source_url(self):
+        installer = get_cloudify.CloudifyInstaller()
+        req_list = installer._get_default_requirement_files(cloudify_cli_url)
+        self.assertEquals(len(req_list), 1)
+        self.assertIn('dev-requirements.txt', req_list[0])
```

Comment:
this can break easily and unexpectedly.

what i suggest instead is helper function such as:

    import os
    import tarfile
    import tempfile

    tempdir = tempfile.mkdtemp()
    fpath = os.path.join(tempdir, 'dev-requirements.txt')
    try:
        f = open(fpath, 'w')
        f.write('something==1\n')
    finally:
        f.close()

    try:
        tar = tarfile.open(name='sample.tar.gz', mode='w:gz')
        tar.add(name=tempdir, arcname='maindir')
    finally:
        tar.close()

this will create dummy tars for test